src/handlers/glacier.rs

```diff
@@ -59,6 +61,47 @@ impl Handler for GlacierHandler {
     }
 }
 
-async fn handle_input(_ctx: &Context, _event: &Event, _cmd: GlacierCommand) -> anyhow::Result<()> {
+async fn handle_input(ctx: &Context, event: &Event, cmd: GlacierCommand) -> anyhow::Result<()> {
+    let is_team_member = if let Err(_) | Ok(false) = event.user().is_team_member(&ctx.github).await
+    {
+        false
+    } else {
+        true
+    };
+
+    if !is_team_member {
+        return Ok(())
+    };
+
+    let url = cmd.source;
+    let response = reqwest::get(&format!("{}{}", url.replace("github", "githubusercontent"), "/playground.rs")).await?;
+    let body = response.text().await?;
+
+    let number = event.issue().unwrap().number;
+    let user = event.user();
+
+    let octocrab = octocrab::instance();
+
+    let fork = octocrab.repos("rust-lang", "glacier");
```

Comment:
I assume ~~we~~ @Mark-Simulacrum will create a fork for the bot, so to not pollute the main `rust-lang/glacier` repository? In that case you just need to set the owner to `"rustbot"` instead of `"rust-lang"`